src/openrct2/world/Entrance.cpp

```diff
@@ -488,11 +380,12 @@ void ride_entrance_exit_remove_ghost()
 {
     if (_currentTrackSelectionFlags & TRACK_SELECTION_FLAG_ENTRANCE_OR_EXIT)
     {
-        game_do_command(
-            gRideEntranceExitGhostPosition.x,
-            (GAME_COMMAND_FLAG_5 | GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED | GAME_COMMAND_FLAG_APPLY),
-            gRideEntranceExitGhostPosition.y, _currentRideIndex, GAME_COMMAND_REMOVE_RIDE_ENTRANCE_OR_EXIT,
+        auto rideEntranceExitRemove = RideEntranceExitRemoveAction(
+            { gRideEntranceExitGhostPosition.x, gRideEntranceExitGhostPosition.y }, _currentRideIndex,
             gRideEntranceExitGhostStationIndex, gRideEntranceExitPlaceType == ENTRANCE_TYPE_RIDE_EXIT);
+
+        rideEntranceExitRemove.SetFlags(GAME_COMMAND_FLAG_5);
```

Comment:
GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED missing

Reply:
I played around with this and came to the conclusion that this is not required.